run-devel.sh

```diff
@@ -50,7 +50,11 @@ fi
 ROS_DISTRO=$ROS_DISTRO \
   VOLUMES_FOLDER=$VOLUMES_FOLDER \
   CONTAINER_NAME=$CONTAINER_NAME \
-  docker-compose -p "$ROS_DISTRO" --env-file .env up $BUILD_IMAGE_OPT -d devel
+  docker-compose -p "$ROS_DISTRO" --env-file .env $BUILD_IMAGE_OPT devel
+ROS_DISTRO=$ROS_DISTRO \
+  VOLUMES_FOLDER=$VOLUMES_FOLDER \
+  CONTAINER_NAME=$CONTAINER_NAME \
+  docker-compose -p "$ROS_DISTRO" --env-file .env up -d devel
```

Comment:
I just want to ask you to check if you have problems running two containers of different distros simultaneously (the -p option is intended to solve this, but I don't know how it behaves by setting it on separate `build`/`pull` and `up` steps)